scripts/ths_r4_sanity.py

```diff
@@ -136,10 +130,19 @@ def get_table_rows(random_args_list):
         assert len(src_lt['branch']) == 1
 
         # print(gsim_lt['uncertainty'])
-
         # source digest
-        srcs = "|".join(sorted(src_lt['branch']['A'].split('|')))
-        src_id = registry.source_registry.get_by_identity(srcs)
+        try:
```

Comment:
@chrisdicaprio this is the `test meta` change. Maybe It could be described better .... I'll change that